client/src/components/ProductDetail/categoryName.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Fragment, useState } from 'react';
+
+let CategoryName = function (props) {
+  return (
+    <Fragment>
+      {props.productInfo.map((productData) => {
+        return (
+          <div key={productData.id}>
+            <div className='product-category'>
+              {' '}
+              Category:{productData.category}
+            </div>
+            <h2 className='expanded-product-name'>{productData.name}</h2>
+          </div>
+        );
+      })}
+      {props.salesPrice === null ? (
```

Comment:
love how you use these ternary operators - i need to do this more too

Reply:
Thanks!  It works at least lol